src/id5-api.js

```diff
@@ -88,15 +104,31 @@ ID5.init = function (options) {
                 utils.logError(error);
               }
             }
-          }, data, { method: 'GET', withCredentials: true });
+          }, JSON.stringify(data), { method: 'POST', withCredentials: true });
```

Comment:
We need here `contentType: "application/json; charset=utf-8", `

Reply:
added as part of https://github.com/id5io/id5-api.js/pull/8/commits/029290c8c0ade11c37bd2631b867eed7a63b61f6